### PR TITLE
Multiple servers & some refactoring

### PR DESCRIPTION
**Pull request by balthazar**
Just a random idea... Nothing to see here heheheheheh....

Apart from the main feature, refactors include switching to indented code to remove two extra newlines when syntax highlight is not needed, a small comment that was untrue, moving things around to prevent duplicate code or improve readability.

I also removed the code responsible to remove the last slash from url's, was unsure about it's real purpose, is this some compatibility issue? I can re-add it if you want.

PS: keep my signed commits please 😭

### CODE REVIEW

File: src/history.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import r from 'superagent'
 import fs from 'fs'
 import path from 'path'
-import _ from 'lodash'
 
 const HISTORY_FILE_NAME = path.join(process.env.HOME, '.samus_history')
 
-// hard coded variable!! baaad!! (ps: I don't care, I'm the only user, lol)
+// hard coded variable!! baaad!! (ps: I don't care, we are two to use this, lol)
```
Comment (meriadec) [inline src/history.js:7]: 😄

**Comment by meriadec** [comment]
I'm liking it, thanks!

Also, for the last slash in urls, I can remember that it caused problems when doing request, so I stripped it. Can you confirm that you have no problem with and without a backslash at the end of your url? I think that we need to be sure not having one, because going into a folder will add a `/` more at the end, so it will have `//` if we keep it in url, right?

**Comment by balthazar** [comment]
Yep, the url have an extra backslash but it work on my end, although the behavior may differ on Arch, and I can't really try it on another thing than OSX right now.

File: src/history.js (hunk 1 above)
Comment (balthazar) [reply src/history.js:7]: We'll need to add this to the `CONTRIBUTING.md`

**Comment by meriadec** [comment]
Thanks for your work, dude

**Pull request merged**